Modules/Sources/JetpackStats/Utilities/Modifiers/CardModifier.swift

```diff
@@ -0,0 +1,19 @@
+import SwiftUI
+
+struct CardModifier: ViewModifier {
```

Comment:
Nitpick: This modifier does not seem to be necessary. I think we can move the function body to `func cardStyle()`?